src/analysis/ConfigsParticleDecomp.hpp

```diff
@@ -147,13 +147,14 @@ namespace espressopp {
         * !! currently only works for particles numbered like 0, 1, 2,... !!
         * !! with each chain consisting particles with subsequent ids     !!
        */
-      ConfigsParticleDecomp(shared_ptr<System> system, int _chainlength): SystemAccess (system){
+      ConfigsParticleDecomp(shared_ptr<System> system, int _chainlength, int _start_pid): SystemAccess (system){
         // by default key = "position", it will store the particle positions
         // (option: "velocity" or "unfolded")
         esutil::Error err(system->comm);
         
         key = "position";
         chainlength = _chainlength;
+	start_pid = _start_pid;
```

Comment:
Indentation?

Reply:
strange ... my editor somehow put <tabs> in the beginning ... I resolved the problem for all subsequent indentations